src/lib/MaskedInput.tsx

```diff
@@ -6,6 +6,7 @@ import { InputProps } from 'antd/lib/input';
 
 export type MaskedInputProps = InputProps & {
   mask: string;
+  showMask?: boolean;
```

Comment:
`showMask: true` would  be the default behaviour as before

Reply:
Good idea!